app/presenters/queries/expanded_link_set.rb

```diff
@@ -3,10 +3,11 @@ module Queries
     class ExpandedLinkSet
       attr_reader :draft
 
-      def initialize(content_id:, draft: false, locale: Edition::DEFAULT_LOCALE)
+      def initialize(content_id: nil, locale: nil, edition: nil, draft: false)
```

Comment:
How about if we make it all more explicit?

So rather than attr_reader for draft etc have those as methods which are like

```
def draft
  edition ? edition.draft? : options.fetch(:draft)
end
```

And maybe initialize via static methods that are either `by_edition` or `by_content_id` (or similar)

Just trying wondering how we can unburden it a bit and cut back on ability to have edition conflict with options